src/model/cloud-runner/workflows/async-workflow.ts

```diff
@@ -0,0 +1,60 @@
+import CloudRunnerSecret from '../services/cloud-runner-secret';
+import CloudRunnerEnvironmentVariable from '../services/cloud-runner-environment-variable';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import { CloudRunnerFolders } from '../services/cloud-runner-folders';
+import CloudRunner from '../cloud-runner';
+
+export class AsyncWorkflow {
+  public static async runAsyncWorkflow(
+    environmentVariables: CloudRunnerEnvironmentVariable[],
+    secrets: CloudRunnerSecret[],
+  ): Promise<string> {
+    try {
+      CloudRunnerLogger.log(`Cloud Runner is running async mode`);
+
+      let output = '';
+
+      output += await CloudRunner.Provider.runTaskInWorkflow(
+        CloudRunner.buildParameters.buildGuid,
+        `ubuntu`,
+        `apt-get update > /dev/null
+apt-get install -y curl tar tree npm git git-lfs jq git > /dev/null
```

Comment:
Would it be easier to read if this was extracted to a shell script as well? Or would there be a way to run the same commands here, but as typescript instead?

Reply:
Will resolve through referenced issue. Might address post-merge.